server/lib/currency.js

```diff
@@ -18,6 +18,21 @@ function getDate(date = 'latest') {
   return date;
 }
 
+export function formatCurrency(currency) {
+  const currencyFormats = {
+    AUD: value => `${value} AUD`,
+    CAD: value => `${value} CAD`,
+    EUR: value => `€${value}`,
+    GBP: value => `£${value}`,
+    INR: value => `₹${value}`,
+    MXN: value => `${value} MXN`,
+    SEK: value => `kr ${value}`,
+    USD: value => `$${value}`,
+    UYU: value => `$U ${value}`,
+  };
+  return currencyFormats[currency];
```

Comment:
Let's take this as an opportunity to improve what we have. Currently if we use this function with an unknown currency it will crash. We can fix it easily with the following:

```suggestion
  return currencyFormats[currency] || value => `${value} ${currency}`;
```

Reply:
Will implement